### DISCUSSION TITLE
Plans to bring Dedicated NDI Output to the new version?

### DISCUSSION THREAD

**Discussion by datlaunchystark** [DistroAV/DistroAV#750] [General]
I tested out Build #30 and the option for "Dedicated NDI Output" is now missing under filters.  Is there plans for that feature to be implemented or is it going to be removed?
https://github.com/Palakis/obs-ndi/actions/runs/2894449748

**Comment by FaultyMusic** [comment]
also would like to know

**Reply by jbcurler2010** [reply]
Also the option to set any NDI Output as HX rather than full NDI.
